### PR TITLE
removed authentication to view  locations and accommodations

### LINKED ISSUE #36: Remove Authentication to retrieve location and accomodation
- it require user to be authenticated to view place and location , and we need to make it open to everyone becouse we are using it on welcome page 
- 
- we want to access popular locations and accomodation without authenticating user

### PR DESCRIPTION
**Pull request by inezabonte**
# Description

I removed the authentication so that anyone can view locations and accommodations without being authenticated

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `ch-setup-testing-env`
- Run the command `npm i`
- Run the command `npm run test` to run tests locally 
- If you push new changes, the test shall run on travis

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)

### CODE REVIEW

File: src/routes/api/accommodationsRoute.js [hunk 1]
```diff
@@ -250,7 +248,7 @@ router.get('/', permit(['view accommodations']), getAccommodations);
  *                      description: Availability of wifi
  *                      example: false
  */
-router.get('/:id', permit(['view accommodations']), getOneAccommodation);
```
Comment (salviosage) [inline src/routes/api/accommodationsRoute.js]: from my understanding this should be protected , as per opening accomodations was for the sake of  landing page . so viewing more details should be protected.

File: src/routes/api/locationsRoute.js [hunk 2]
```diff
@@ -138,7 +134,7 @@ router.get('/', permit(['view locations']), getLocations);
  *
  *
  */
-router.get('/:id', permit(['view locations']), getOneLocation);
+router.get('/:id', getOneLocation);
```
Comment (salviosage) [inline src/routes/api/locationsRoute.js]: the same here , please keep plotecting this route

**Review by salviosage — COMMENTED** [review]
Nice work , i left some comments

File: src/routes/api/accommodationsRoute.js (hunk 1 above)
Comment (inezabonte) [reply src/routes/api/accommodationsRoute.js]: @salviosage fixed ✅

File: src/routes/api/locationsRoute.js (hunk 2 above)
Comment (inezabonte) [reply src/routes/api/locationsRoute.js]: @salviosage fixed ✅

**Review by salviosage — APPROVED** [review]
LGTM

**Pull request merged**